tests/integ/kms_utils.py

```diff
@@ -12,19 +12,25 @@
 # language governing permissions and limitations under the License.
 from __future__ import absolute_import
 
+import json
+
 from botocore import exceptions
 
-KEY_ALIAS = "SageMakerIntegTestKmsKey"
+PRINCIPAL_TEMPLATE = '["{account_id}", "{role_arn}", "arn:aws:iam::{account_id}:role/{' \
+                     'sagemaker_role}"] '
+
+KEY_ALIAS = "SageMakerKmsKeyIntegTest"
+POLICY_NAME = "default"
 KEY_POLICY = '''
 {{
   "Version": "2012-10-17",
-  "Id": "sagemaker-kms-integ-test-policy",
+  "Id": "{id}",
   "Statement": [
     {{
       "Sid": "Enable IAM User Permissions",
       "Effect": "Allow",
       "Principal": {{
-        "AWS": "*"
+        "AWS": {principal}
```

Comment:
why not grant to account root instead of a role? that way any entity in the account should get these permissions.

Reply:
Granting to the account was my first attempt, unfortunately KMS permissions don't propagate to the assumed role when we do that.

Reply:
Btw, the account root is already one of the principals in the list.